### PR TITLE
updated branch (#35)

### PR DESCRIPTION
**Pull request by gkgbunny**
https://github.com/nodexpertsdev/RnD/issues/35#issue-432544682

### CODE REVIEW

File: expressEngine/lib/parser/parser.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export default (controller = null, functionName = '') => async (req, res) => {
       throw new Error(error.functionNotFound);
     }
 
+    console.log("inside Parser");
```
Comment (mayank-nx) [inline expressEngine/lib/parser/parser.js]: @gkgbunny Remove console.log

File: expressEngine/lib/validationHandler/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import validateRequired from './validateRequired';
+
+export {
+	validateRequired,
```
Comment (mayank-nx) [inline expressEngine/lib/validationHandler/index.js]: @gkgbunny Incorrect indentation. Fix it and Enable eslint for your editor

File: expressEngine/product/service/product.js [hunk 3]
```diff
@@ -7,13 +7,16 @@ import { Product, ProductDetail } from '../../model';
 // import messages
 import {success} from '../../cms/product';
 
+//import validateRequired
+import { validateRequired } from '../../lib/validationHandler';
+
 class Service extends BaseService {
   registerProduct = async (data) => {
     try {
       const {name, supplierId,unitPrice, productPackage, isDiscontinued  } = data,
```
Comment (DPDeepak) [inline expressEngine/product/service/product.js]: const {name, supplierId,unitPrice, productPackage, isDiscontinued  } = data,
fix eslint error.
is should be
const { name, supplierId,unitPrice, productPackage, isDiscontinued } = data,

Comment (gkgbunny) [reply expressEngine/product/service/product.js]: you also forgot to add space between supplierId and unitPrice

File: expressEngine/lib/validationHandler/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import validateRequired from './validateRequired';
+
+export { validateRequired };
```
Comment (aman-successive) [inline expressEngine/lib/validationHandler/index.js]: The name of the function is "validateReqData" in validationRequired.js and "validationRequired" in index.js.
Please export and import using the same name

Comment (gkgbunny) [reply expressEngine/lib/validationHandler/index.js]: I have updated now

**Comment by aman-successive** [comment]
Please run the User module on Postman, I think there might be a problem in the flow of your code

**Comment by gkgbunny** [comment]
I have resolved it now

**Pull request merged**